src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -246,6 +254,7 @@
         (PegasusConfig, TFPegasusModel),
         (BlenderbotConfig, TFBlenderbotModel),
         (BlenderbotSmallConfig, TFBlenderbotSmallModel),
+        (LayoutLMConfig, TFLayoutLMModel),
```

Comment:
```suggestion
```
It's already defined up there.

Reply:
fixed [here](https://github.com/huggingface/transformers/pull/10636/commits/3bee70daf2a71066335d360761ce5c7bb432500a)